metals/src/main/scala/scala/meta/metals/MetalsServices.scala

```diff
@@ -441,6 +443,22 @@ class MetalsServices(
   }
   private def sbtExec(): Unit = sbtExec(latestConfig().sbt.command)
 
+  private val loadPluginJars: Coeval[List[AbsolutePath]] = Coeval.evalOnce {
+    Jars.fetch("ch.epfl.scala", "load-plugin_2.12", "0.1.0+2-496ac670")
+  }
+
+  private def sbtExecWithMetalsPlugin(commands: String*): Unit = {
+    val metalsPluginModule =
+      ModuleID("org.scalameta", "sbt-metals", "0.1.0-M1+84-a3ffb91c")
```

Comment:
Maybe use Buildinfo to use the current version here?

Reply:
Right, I thought about it, but then forgot

Reply:
Done: dde1600